bindings/node/src/Version.cpp

```diff
@@ -0,0 +1,140 @@
+#include <arbiter/Version.h>
+#include "Version.h"
+
+namespace ArbiterNodeBindings {
+
+using v8::Context;
+using v8::Function;
+using v8::FunctionCallbackInfo;
+using v8::FunctionTemplate;
+using v8::Isolate;
+using v8::Just;
+using v8::Local;
+using v8::Maybe;
+using v8::Nothing;
+using v8::Number;
+using v8::Object;
+using v8::Persistent;
+using v8::String;
+using v8::Value;
+
+Persistent<Function> SemanticVersion::constructor;
+
+SemanticVersion::SemanticVersion(unsigned major, unsigned minor, unsigned patch, Maybe<char *> prereleaseVersion, Maybe<char *> buildMetadata) {
+  _semanticVersion = ArbiterCreateSemanticVersion(
+    major,
+    minor,
+    patch,
+    (prereleaseVersion.IsJust() ? prereleaseVersion.FromJust() : NULL),
+    (buildMetadata.IsJust() ? buildMetadata.FromJust() : NULL)
+  );
+}
+
+SemanticVersion::SemanticVersion(const char *string) {
+  _semanticVersion = ArbiterCreateSemanticVersionFromString(string);
+}
+
+SemanticVersion::~SemanticVersion() {
```

Comment:
I think you should use `ArbiterFree()` here, like @kastiglione suggested, as long as you store the semantic version pointer as non-const (which is correct, since you own it), and explicitly cast here with `static_cast<void *>(…)`.